src/MizarData/include/MizarData/ActiveFunctionTimePerFrameComparator.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_STATISTICS_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+#define MIZAR_STATISTICS_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+
+#include <utility>
+
+#include "ClientData/ScopeStats.h"
+#include "MizarData/BaselineOrComparison.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+#include "Statistics/Gaussian.h"
+#include "Statistics/StatisticsUtils.h"
+
+namespace orbit_mizar_data {
+
+// TODO(b/236130122) it should be moved to a new library
+template <typename Counts, typename FrameTrackStats>
+class ActiveFunctionTimePerFrameComparatorTmpl {
```

Comment:
Does `Tmpl` stand for template? I don't think we should abbreviate like that.

Reply:
In defence of Tmpl: 
1. It's a commonly used suffix
2. It's the `Impl`'s suffix ~~evil~~templated twin. 

Is the problem solely with removing the letters from the middle of the resulted abbreviation? Would `Templ` be fine?

Reply:
As discussed, if this is a common suffix in this technique, then just keep it as it is.